### ISSUE TITLE
[SDK 50] `new NativeEventEmitter()` warning when using expo-av

### ISSUE THREAD

**Issue by victormamede** [expo/expo#26592]
### Minimal reproducible example

import { Audio } from "expo-av";

### Summary

```
`new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method.
```

This warning was thrown after upgrading my project to expo 50. Upon further investigation I realized that it would go away after removing the import for `expo-av@~13.10.3` which leads me to believe that something in this package is not yet fully compatible with expo 50

### Environment

```
expo-env-info 1.2.0 environment info:
    System:
      OS: Linux 6.7 Arch Linux
      Shell: 5.9 - /usr/bin/zsh
    Binaries:
      Node: 21.6.0 - /usr/bin/node
      Yarn: 1.22.21 - /usr/bin/yarn
      npm: 10.3.0 - /usr/bin/npm
    IDEs:
      Android Studio: AI-231.9392.1.2311.11255304
    npmPackages:
      expo: ^50.0.0 => 50.0.2
      expo-router: ~3.4.3 => 3.4.3
      react: 18.2.0 => 18.2.0
      react-native: 0.73.2 => 0.73.2
    npmGlobalPackages:
      eas-cli: 7.0.0
    Expo Workflow: managed
```

**Comment by lluisandreu** [comment]
I can reproduce this error after upgrading to sdk 50.

**Comment by miguelquispe** [comment]
I encountered an issue with my project after upgrading from SDK 48 to SDK 50 and updating expo-av from version 13.2.1 to version 13.10.4 as per the instructions provided in the [E**xpo SDK Upgrade** documentation](https://docs.expo.dev/workflow/upgrading-expo-sdk-walkthrough/). Upon running npx expo start, a warning regarding **new NativeEventEmitter()** appeared.

To resolve this issue, I had to downgrade expo-av back to version 13.2.1.
